app/src/main/java/io/intrepid/contest/models/Participant.java

```diff
@@ -0,0 +1,29 @@
+package io.intrepid.contest.models;
+
+import java.util.Date;
+import java.util.UUID;
+
+public class Participant {
+    private UUID id;
```

Comment:
I added them because they'd be automatically filled in by Retrofit, but I removed them until they're used then!